include/samurai/field.hpp

```diff
@@ -1188,14 +1188,15 @@ namespace samurai
     inline auto ScalarField<mesh_t, value_t>::begin() -> iterator
     {
         using mesh_id_t = typename mesh_t::mesh_id_t;
-        return iterator(this, this->mesh()[mesh_id_t::cells].begin());
+        // add comment : the CI is buggy ahahaha
+        return iterator(this, std::as_const(this->mesh()[mesh_id_t::cells]).begin());
```

Comment:
Could you replace `std::as_const(this->mesh()[mesh_id_t::cells]).begin()` by `this->mesh()[mesh_id_t::cells].cbegin()` ?